src/Methods/Address/GetSpendables.ts

```diff
@@ -74,7 +95,7 @@ export default method({
       });
 
       totalValue += output.value;
-      if (totalValue >= value || spendables.length >= MAX_SPENDABLES) {
```

Comment:
Can we leave this in? This checks if the `totalValue` of the scanned UTXOs exceeds the `value` requested.